### DISCUSSION TITLE
The kernel appears to have died

### DISCUSSION THREAD

**Discussion by Recep1997** [autogluon/autogluon#1804] [Q&A]
Hello,

I have reproduced the following tutorial a week ago. 

https://auto.gluon.ai/stable/tutorials/text_prediction/beginner.html

But after a week the same code somehow doesn't work anymore. It comes the message that the kernel appears to have died. 

I tried to update all imports, but nothing changed :(

**Comment by sxjscience** [comment]
It might be related to https://github.com/awslabs/autogluon/issues/1762 . Can you submit an issue instead so we can track it? Which type of machine are you using?

**Reply by Recep1997** [reply]
I use my MacBook and Jupyter from Anaconda. I think it is because of Tensorflow. Two weeks ago the same code worked.

**Reply by sxjscience** [reply]
What error are you getting?

**Reply by Recep1997** [reply]
My Kernal instantly dies, if i use the fit() Method from AutoGluon

**Reply by sxjscience** [reply]
This is new to me. Can you copy the codes to a script and run the script instead?

**Reply by sxjscience** [reply]
Have you updated any other packages locally? I recommend to run a diagnostic command: 

```python
# Replace this code with the output of the following:
from autogluon.core.utils import show_versions
show_versions()
```

**Reply by Recep1997** [reply]
INSTALLED VERSIONS
------------------
date                 : 2022-06-11
time                 : 22:51:34.654660
python               : 3.8.13.final.0
OS                   : Darwin
OS-release           : 21.5.0
Version              : Darwin Kernel Version 21.5.0: Tue Apr 26 21:08:22 PDT 2022; root:xnu-8020.121.3~4/RELEASE_X86_64
machine              : x86_64
processor            : i386
num_cores            : 4
cpu_ram_mb           : 8192
cuda version         : None
num_gpus             : 0
gpu_ram_mb           : []
avail_disk_size_mb   : 43878

autogluon.common     : 0.4.2
autogluon.core       : 0.4.2
autogluon.features   : 0.4.2
autogluon.tabular    : 0.4.2
autogluon.text       : 0.4.2
autogluon.vision     : 0.4.2
autogluon_contrib_nlp: None
boto3                : 1.24.2
catboost             : 1.0.6
dask                 : 2021.11.2
distributed          : 2021.11.2
fairscale            : None
fastai               : 2.5.6
gluoncv              : 0.11.0
lightgbm             : 3.3.2
matplotlib           : 3.5.1
networkx             : 2.7.1
nptyping             : 1.4.4
numpy                : 1.21.5
omegaconf            : 2.1.2
pandas               : 1.3.5
PIL                  : 9.0.1
protobuf             : None
psutil               : 5.8.0
pytorch_lightning    : 1.6.4
ray                  : 1.10.0
requests             : 2.27.1
scipy                : 1.7.3
sentencepiece        : None
setuptools           : 59.5.0
skimage              : 0.19.2
sklearn              : 1.0.2
smart_open           : 5.2.1
timm                 : 0.5.4
torch                : 1.10.2
torchmetrics         : 0.7.3
tqdm                 : 4.64.0
transformers         : 4.16.2
xgboost              : 1.4.2
